ATTRIBUTION.md

```diff
@@ -17,6 +17,108 @@ This package contains a variety of materials, including data sets and related ma
 * Location: ***prereise/gather/data/ba_to_county.txt***
 
 
+#### US County Map Shape Files
+#### Source
+* Name: US County map
+* Author: US Census Bureau
+* Description: US county map
+* Source: https://www.census.gov/
+* Exact source location: https://www.census.gov/geographies/mapping-files/time-series/geo/cartographic-boundary.html
+#### Destination
+* Modifications to source files(s): None
+* Location: ***prereise/gather/data/cb_2020_us_county_500k.zip***
+
+
+#### Retail Service Territory 
+#### Source
+* Name: Electric Retail Service Territory 
+* Author: Homeland Infrastructure Foundation-Level Data (HIFLD)
+* Description: electric retail service territory
+* Source: https://hifld-geoplatform.opendata.arcgis.com/
+* Exact source location: https://hifld-geoplatform.opendata.arcgis.com/datasets/private-non-retail-shipping-facilities/about
+#### Destination
+* Modifications to source files(s): none
+* Location: ***prereise/gather/data/remap_ba_area/cb_2020_us_county_500k.zip***
+
+
+#### Subregions of NYISO
+#### Source
+* Name: subregions of NYISO
+* Author: ArcGIS
+* Description: subregions of NYISO
+* Source: https://www.arcgis.com/
+* Exact source location: https://www.arcgis.com/home/item.html?id=6fd1de467b134f47a607721f23a69f0c
+#### Destination
+* Modifications to source files(s): None for A-F, they are directly downloaded by the program; G_I are made by grouping and splitting county shapes
+* Location: ***prereise/gather/data/remap_ba_area/nyiso2/***
+
+#### CAISO
+#### Source
+* Name: subregions of CAISO
+* Author: CAISO
+* Description: the subregions of CAISO
+* Source: https://www.caiso.com/
+* Exact source location: https://www.caiso.com/Documents/ISO_ContingencyModelingProposal-SCE_Presentation.pdf
+#### Destination
+* Modifications to source files(s): select county shapes from the image map
+* Location: ***prereise/gather/data/remap_ba_area/CA_balancing_authorities/***
```

Comment:
These files appear to be the ones that I originally put into the dropbox folder, which came from https://cecgis-caenergy.opendata.arcgis.com/datasets/CAEnergy::california-electric-balancing-authority-areas/, not from any conversion from CAISO documents.